uwreact_bringup/launch/zeds.launch

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+
+<launch>
+    <arg name="zed_id1"               default="0" />
+
+    <arg name="pose_frame"            default="map" />
+    <arg name="camera_model"          default="0" /> <!-- 0=ZED, 1=ZEDM-->
+    <arg name="serial_number"         default="0" />
+    <arg name="verbose"               default="true" />
+
+    <arg name="resolution"            default="2" /> <!--0=RESOLUTION_HD2K, 1=RESOLUTION_HD1080, 2=RESOLUTION_HD720, 3=RESOLUTION_VGA -->
+    <arg name="frame_rate"            default="30" />
+    <!-- RESOLUTION_HD2K   -> 2208*1242, available framerates: 15 fps.
+         RESOLUTION_HD1080 -> 1920*1080, available framerates: 15, 30 fps.
+         RESOLUTION_HD720  -> 1280*720,  available framerates: 15, 30, 60 fps.
+         RESOLUTION_VGA    -> 672*376,   available framerates: 15, 30, 60, 100 fps. -->
+
+    <arg name="depth_topic"           default="depth/depth_registered" />
+    <arg name="depth_cam_info_topic"  default="depth/camera_info" />
+    <arg name="laserscan_topic"       default="depth/laserscan"/>
+
+    <arg name="base_frame"            default="zed_camera_center"/>
+    <arg name="laserscan_height"      default="10"/>
+    <arg name="range_min"             default="0.45"/>	
+
+    <!-- First ZED camera -->
+    <group ns="zed1">
+        <include file="$(find zed_wrapper)/launch/zed_camera.launch">
+            <arg name="zed_id"                value="$(arg zed_id1)" />
+
+            <arg name="pose_frame"            value="$(arg pose_frame)" />
+            <arg name="camera_model"          value="$(arg camera_model)" />
+            <arg name="serial_number"         default="$(arg serial_number)" />
+            <arg name="resolution"            default="$(arg resolution)" />
+            <arg name="frame_rate"            default="$(arg frame_rate)" />
+            <arg name="verbose"               default="$(arg verbose)" />
+
+            <arg name="depth_topic"           default="$(arg depth_topic)" />
+            <arg name="depth_cam_info_topic"  default="$(arg depth_cam_info_topic)" />
+            <arg name="base_frame"            default="$(arg base_frame)" />
+        </include>
+
+        <node name="depthimage_to_laserscan" pkg="depthimage_to_laserscan" type="depthimage_to_laserscan">
+            <remap from="image"            to="$(arg depth_topic)" />
+            <remap from="camera_info"      to="$(arg depth_cam_info_topic)" />
+            <remap from="scan"             to="$(arg laserscan_topic)"/>
+
+            <param name="output_frame_id"  value="$(arg base_frame)" />
+            <param name="scan_height"      value="$(arg laserscan_height)"/>
+            <param name="range_min"        value="$(arg range_min)"/>
+        </node>
+    </group>
```

Comment:
This or should include the launch of the second camera imo

Reply:
I was gonna look into combining the depth/laserscan data from both zeds together next, so I was just gonna add the second one with that feature if its all the same to you (also I there aren't enough usbs on the jetson to plug in two zeds and the keyboard since remoting in is rip)